src/__tests__/utils.js

```diff
@@ -51,6 +51,25 @@ test(`resolveBin resolves to the binary if it's in $PATH`, () => {
   expect(whichSyncMock).toHaveBeenCalledWith('cross-env')
 })
 
+describe('for windows', () => {
+  let realpathSync
```

Comment:
Describe block used so we don't need to mock fs for other tests that rely on real fs